package.json

```diff
@@ -383,6 +383,12 @@
                 }
             ]
         },
+        "jsonValidation": [
+            {
+                "fileMatch": "staticwebapp.config.json",
```

Comment:
Change this to 
`/staticwebapp.config.json`

Eric has it this way in Functions, I think it's so it doesn't _have_ to be in the root.